second_test/package.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Hmm not sure why you put that in. You should have a single `package.json` file for a Javascript project and in order to alter the spec files you want to run, you should alter the command in your `script` object (maybe with a wildcard or the use of suites).